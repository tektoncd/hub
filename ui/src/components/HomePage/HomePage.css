.hub-banner-text{
      text-align: center;
}