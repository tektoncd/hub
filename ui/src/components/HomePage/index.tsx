import React from 'react';
import { Grid, GridItem, PageSection } from '@patternfly/react-core';
import LeftPane from '../LeftPane';
import Background from '../Background';
import Resources from '../../containers/Resources';

const HomePage = () => {
  return (
    <>
      <Background />
      <PageSection>
        <Grid hasGutter>
          <GridItem span={2}>
            <LeftPane />
          </GridItem>
          <GridItem span={10} rowSpan={1}>
            <Resources />
          </GridItem>
        </Grid>
      </PageSection>
    </>
  );
};

export default HomePage;
