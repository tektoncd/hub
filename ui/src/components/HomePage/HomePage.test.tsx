import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '.';

it('should render the home page component', () => {
  const component = shallow(<HomePage />);
  expect(component.debug()).toMatchSnapshot();
});
