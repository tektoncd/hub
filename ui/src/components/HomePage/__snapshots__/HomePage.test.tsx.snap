// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home page component 1`] = `
"<Fragment>
  <Background />
  <PageSection>
    <Grid hasGutter={true}>
      <GridItem span={2}>
        <Memo() />
      </GridItem>
      <GridItem span={10} rowSpan={1}>
        <Memo() />
      </GridItem>
    </Grid>
  </PageSection>
</Fragment>"
`;
